etc/systemd/system/worker.duffy.service

```diff
@@ -0,0 +1,13 @@
+[Unit]
+After=network.target
+
+[Service]
+User=duffy
+Group=duffy
+ExecStart=/venv/bin/duffy -c /etc/duffy-example-config.yaml worker
```

Comment:
This needs to be `... worker -B --schedule-filename ...`. The `-B` option tells Celery to run the "beat" component, i.e. service periodic tasks, `--schedule-filename ...` tells it the path to a (writable) file where it stores scheduled tasks.